src/sources/azure_event_hubs/config.rs

```diff
@@ -0,0 +1,113 @@
+use vector_config::configurable_component;
+use vector_core::config::LogNamespace;
+
+use codecs::decoding::{DeserializerConfig, FramingConfig};
+
+use crate::{
+    config::{AcknowledgementsConfig, Output, SourceConfig, SourceContext},
+    kafka::{KafkaAuthConfig, KafkaSaslConfig},
+    serde::{bool_or_struct, default_decoding, default_framing_message_based},
+    sources::kafka::{
+        default_auto_offset_reset, default_commit_interval_ms, default_fetch_wait_max_ms,
+        default_headers_key, default_key_field, default_offset_key, default_partition_key,
+        default_session_timeout_ms, default_socket_timeout_ms, default_topic_key,
+        KafkaSourceConfig,
+    },
+    tls::TlsEnableableConfig,
+};
+
+/// Configuration for the `azure_event_hubs` source.
+/// This component is a simple wrapper over the `kafka` source.
+/// See the documentation [here](https://docs.microsoft.com/en-gb/azure/event-hubs/event-hubs-for-kafka-ecosystem-overview)
+/// for details on how `azure_event_hubs` can use `kafka`.
```

Comment:
```suggestion
/// See the documentation [here][event_hubs_docs] for details on how `azure_event_hubs` can use `kafka`.
///
///[event_hubs_docs]: https://docs.microsoft.com/en-gb/azure/event-hubs/event-hubs-for-kafka-ecosystem-overview
```

I'd use this format instead - [ref](https://github.com/vectordotdev/vector/blob/master/src/sinks/kafka/config.rs#L67-L71)